src/um/winbase.rs

```diff
@@ -1249,7 +1249,7 @@ extern "system" {
         hUpdate: HANDLE,
         fDiscard: BOOL
     ) -> BOOL;
-    #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
+    #[cfg(any(target_arch = "x86", target_arch = "x86_64", target_arch = "aarch64"))]
```

Comment:
What architecture is this method *not* supported on?

Reply:
My copy of `winbase.h` suggests this is supported everywhere.  Will remove the conditional.